pyfstat/__init__.py

```diff
@@ -34,6 +34,7 @@
     DMoff_NO_SPIN,
     SliceGridSearch,
     TransientGridSearch,
+    SearchOverGridFile,
```

Comment:
An alternative solution, possibly better, would be to give the existing classes alternative constructors.